.github/workflows/upload-release-to-plugin-repo.yml

```
@@ -267,6 +267,10 @@ jobs:
                   svn checkout "$PLUGIN_REPO_URL/assets" assets \
                   --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
 
+            - name: Delete everything
+              working-directory: ./assets
+              run: find . -maxdepth 1 -not -name ".svn" -not -name "." -not -name ".." -exec rm -rf {} +
```

How about removing all _files_ in the assets directory?
```suggestion
            - name: Delete everything
              run: find assets -type f -remove
```

---

Hmm. I do see a `.svn` folder in my local checkout of the `assets` folder though, and I think we want to retain that, as it contains the necessary metadata for svn to work 🤔 

(FWIW, I copied that step from the `upload` job from the same file, where we're doing pretty much the same to the `trunk` folder.)

---

Oh. I kinda missed the _files_ part, even though you put it in italics 🤦‍♂️ 

Is that not going to descend into the `.svn`folder?

---

Good catch, it would. Find can exclude that path:

```suggestion
            - name: Delete everything
              run: find assets -type f -not -path 'assets/.svn/*' -remove
```

---

Thank you! 

I tried testing this locally; looks like it's `-delete`, not `-remove` 😬 

Other than that, this seems to be working 👍

---

https://github.com/WordPress/gutenberg/pull/68052/commits/aec7218ed3210a1652cb255744d05da849495e7a

---

> I tried testing this locally; looks like it's -delete, not -remove 😬

You are correct sir 🤦